CHANGELOG.md

```diff
@@ -7,6 +7,24 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
 
+### Added
+- Swift Package Manager support
+- TruexAdRenderer support for tvOS (previously iOS only)
+
+### Changed
+- **Migrated from CocoaPods to Swift Package Manager**
+- **Changed library name from `BitmovinYospaceModule` to `BitmovinYospacePlayer`**
```

Comment:
Nit:
```suggestion
- **Changed library name from `BitmovinYospaceModule` to `BitmovinYospacePlayer`**
  This requires changing the import from `import BitmovinYospaceModule` to `import BitmovinYospacePlayer`
```

alternatively (nesting ``` doesn't seem to work nicely 😓 but I guess you get the idea):

```
- **Changed library name from `BitmovinYospaceModule` to `BitmovinYospacePlayer`**
  This requires changing the import:
  ```swift
  // Before
  import BitmovinYospaceModule
  // Now
  import BitmovinYospacePlayer
  ```